packages/manager-react-components/src/components/notifications/useNotifications.ts

```diff
@@ -61,7 +68,12 @@ export const useNotifications = create<NotificationState>((set, get) => ({
         ({ uid }) => uid !== toRemoveUid,
       ),
     })),
-  clearNotifications: () => set(() => ({ notifications: [] })),
+  clearNotifications: () =>
+    set((state) => ({
+      notifications: state.notifications.filter(
+        (notification) => Date.now() - notification.creationTimestamp < 1000,
```

Comment:
```suggestion
        (notification) => Date.now() - notification.creationTimestamp < DEFAULT_MINIMUN_LIFETIME,
```

Optionnal suggestion : Possible to add constant for default minimum lifetime and maybe add parameters if we want more or less on the future ?

Reply:
To me the whole api should be reworked, for example we should be able to set "clearAfterRead" per notification instead of for all of them, but I think it should be another subject, right now I'm just tried to be up to date with MRC v1 on this issue

Reply:
FYI we opened this PR to sync the changes we introduced in MRC 1.x : https://github.com/ovh/manager/pull/14170

Reply:
Oh ok wasn't aware of that I'll just close this one then